person_capture/gui_app.py

```
@@ -3375,8 +3382,60 @@ def _saver():
                             save_q.task_done()
                             break
 
-                        img_path, img, row = item
-                        ok, why = _atomic_jpeg_write(img, img_path, jpg_q)
+                        if isinstance(item, dict) and item.get("type") == "hdr_archive":
+                            try:
+                                frame_pts_sec = item.get("frame_pts_sec")
+                                if frame_pts_sec is not None:
+                                    try:
+                                        frame_pts_sec = float(frame_pts_sec)
+                                    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)